lib/shelley/bench/restore-bench.hs

```diff
@@ -15,6 +15,7 @@
 {-# LANGUAGE TypeApplications #-}
 
 {-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# LANGUAGE BlockArguments #-}
```

Comment:
```suggestion
```
This extension changes the idiom ("dialect") of Haskell that we use as a team, which implies that we want to turn it on either for _all_ modules, or for _none_. That said, we don't have a good process for grooming our idiom, but I intend to improve that.

(Personally, I see little value in this extension.)

Reply:
Ok, I will avoid it here and next and ask to allow it everywhere in the next dev meeting
Personally, I see a huge value in this extension, related to avoiding `lets` and `wheres` as they make hls type inference less useful. Also I see little value in parenthesizing arguments.